book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :author, :title, :rentals
+
+  def initialize(title, author, _rentals)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  def add_rental(rental)
+    @rentals << rental
```

Comment:
This method should be able to create a new rental and add it. It should take `book` itself as a parameter, along with `date` and `person` it belongs to. Something like this. Kindly update your code accordingly.
```
def add_rental(person, date)
    Rental.new(date, self, person)
 end
```